sonar-ballerina-plugin/src/main/java/io/ballerina/sonar/BallerinaChecks.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.ballerina.sonar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BallerinaChecks {
+    static final List<String> CORE_RULES = new ArrayList<>();
+
+    static {
+        CORE_RULES.add("B1");
```

Comment:
What's B1 btw? Didn't we change this?

Reply:
The rule keys provided should match against the corresponding [HTML and JSON files](https://github.com/SonarSource/sonar-analyzer-commons/blob/master/commons/src/main/java/org/sonarsource/analyzer/commons/RuleMetadataLoader.java#L157) for SonarQube to load them. Since it's not possible to create rule files as `ballerina:1.html` and `ballerina:1.json`, consideration was given to name them in this format.

Reply:
Why not Ballerina1?